.pre-commit-config.yaml

```diff
@@ -4,4 +4,18 @@ repos:
     rev: 2.1.1
     hooks:
     -   id: sqlfluff-lint
-    -   id: sqlfluff-fix
\ No newline at end of file
+    -   id: sqlfluff-fix
+  - repo: https://github.com/tconbeer/sqlfmt
+    rev: v0.19.2
+    hooks:
+      - id: sqlfmt
+        entry: >-
+          sqlfmt -v dbt
+            --exclude dbt/venv/**/*
+            --exclude dbt/models/**/*
+            --exclude dbt/target/**/*
+            --exclude dbt/dbt_packages/**/*
+            --exclude dbt/dbt_modules/**/*
```

Comment:
Annoyingly, these `--exclude` options can't be combined via the CLI, so each pattern requires its own `--exclude` flag ([docs](https://docs.sqlfmt.com/getting-started/configuring-sqlfmt#configuration-reference)). We could combine them if we used a `pyproject.toml` file to configure sqlfmt, but that seemed like overkill for one config value; still, I could be convinced to do that if this command seems unwieldy.

Reply:
I think this is totally fine, no need for additional files.